App/Seller/Sources/Seller/UI/MessagesView/MyMessagesView.swift

```diff
@@ -0,0 +1,194 @@
+// system
+import SwiftUI
+
+// Internal
+
+// 3rd
+import OptionalAPI
+
+
+protocol SizingFrameType {
+    
+    static var instance: SizingFrameType { get }
+    
+    var size: CGSize { get }
+}
+
+#if os(macOS)
+import AppKit
+public typealias PlatformScreen = NSScreen
+
+extension NSScreen: SizingFrameType {
+    
+    static var instance: SizingFrameType {
+        NSScreen.main!
+    }
+    
+    public var size: CGSize {
+        frame.size
+    }
+}
+
+#else
+import UIKit
+public typealias PlatformScreen = UIScreen
+
+extension UIScreen: SizingFrameType {
+    static var instance: SizingFrameType {
+        UIScreen.main
+    }
+    
+    public var size: CGSize {
+        bounds.size
+    }
+}
+
+#endif
+
+struct MessagesView: View {
```

Comment:
Would be nice to extract those views to files :)

Reply:
Extracted :D